src/main/java/com/depromeet/todolist/controller/TodoListController.java

```diff
@@ -0,0 +1,55 @@
+package com.depromeet.todolist.controller;
+
+import com.depromeet.todolist.dto.request.RequestTodoDto;
+import com.depromeet.todolist.dto.response.ResponseTodoDto;
+import com.depromeet.todolist.service.TodoService;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.net.URI;
+import java.util.List;
+
+@Slf4j
+@RestController
+@RequestMapping("/api/v1/users/{userId}/todos")
+@RequiredArgsConstructor
+public class TodoListController {
+    private final TodoService todoService;
+
+
+    @GetMapping
+    public ResponseEntity<List<ResponseTodoDto>> userTodoList(@PathVariable String userId) {
+        List<ResponseTodoDto> userTodoList = todoService.getUserTodoList(userId);
```

Comment:
```suggestion
        List<ResponseTodoDto> response = todoService.getUserTodoList(userId);
```
list를 변수명으로 사용하는건 좋지 않아요
자료구조명칭을 네이밍으로 사용하지 않거든요